ykclient.h

```diff
@@ -107,6 +109,7 @@ extern "C"
  */
   extern void ykclient_set_nonce (ykclient_t * ykc, char *nonce);
 
+  extern void ykclient_set_max_retries (ykclient_t * ykc, int retries);
```

Comment:
since a new function is added to the API we need to update libtool versioning in configure.ac (LT_CURRENT, LT_REVISION & LT_AGE)

Reply:
Also.. for me the project doesn't build. For some reason the travis results doesn't show here in the PR but https://travis-ci.org/github/Yubico/yubico-c-client is the build. What errors here is that this new function doesn't get exported on GNU systems, you need to add it to libykclient.map look at how new blocks look like for old additions.

Reply:
Updated `libykclient.map` and incremented the libtool versioning (please take a look to verify they were incremented properly..).